### PR TITLE
Update BBEdit to 12.5

### PR DESCRIPTION
**Pull request by chrisfinazzo**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/bbedit.rb [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
   url "https://s3.amazonaws.com/BBSW-download/BBEdit_#{version}.dmg"
-  appcast 'https://versioncheck.barebones.com/BBEdit.xml'
-  name 'BBEdit'
-  homepage 'https://www.barebones.com/products/bbedit/'
+  appcast "https://versioncheck.barebones.com/BBEdit.xml"
+  name "BBEdit"
+  homepage "https://www.barebones.com/products/bbedit/"
 
-  depends_on macos: '>= :el_capitan'
+  depends_on :macos => ">= :sierra"
```
Comment (reitermarkus) [inline Casks/bbedit.rb]: @MikeMcQuaid, `brew cask style` is not reporting this anymore, it should be `depends_on macos: '>= :sierra'`.

Comment (reitermarkus) [reply Casks/bbedit.rb]: It apparently also corrected the all single to double quotes.

Comment (chrisfinazzo) [reply Casks/bbedit.rb]: @reitermarkus Good to see someone else notice this - honestly would have expected it to be defined in a `.rubocop.yml`.

Comment (MikeMcQuaid) [reply Casks/bbedit.rb]: @reitermarkus I don't think it's possible with the current setup to enforce contradictory styles for Homebrew/brew and formulae vs. cask styles. I added a TODO to investigate getting them closer in sync but for now to get `brew cask style` passing everywhere Casks have just been excluded from contradictory rules.

Comment (reitermarkus) [reply Casks/bbedit.rb]: The best way I could think of would be to have a `.rubocop.yml` in each tap with

```yaml
inherit_from: ../../Homebrew/.rubocop_cask.yml
```

or

```yaml
inherit_from: ../../Homebrew/.rubocop_formula.yml
```

This way the Formula rules also don't conflict with the core code.

Comment (MikeMcQuaid) [reply Casks/bbedit.rb]: @reitermarkus Seems like the only option for now although obviously the duplication is regrettable. Long-term I think we should aim to have the styles more in sync, though.

Comment (core-code) [reply Casks/bbedit.rb]: is there anything preventing this PR from being merged?

Comment (core-code) [reply Casks/bbedit.rb]: i think we should just use the style that any and all other casks use? or is there something i don't get?

Comment (reitermarkus) [reply Casks/bbedit.rb]: > this should probably be using `else` as opposed to `elsif` 

Yes, use `else`.

> Other than that, the only thing is whether Rubocop’s changes should be reverted.

Yes, please revert these.

Comment (chrisfinazzo) [reply Casks/bbedit.rb]: OK, will update.

**Comment by chrisfinazzo** [comment]
Resolved in #54812. I'll open a separate PR for the if-else statement.

**Comment by core-code** [comment]
thanks @chrisfinazzo i already did so in https://github.com/Homebrew/homebrew-cask/pull/54814

**Pull request closed without merging**